viewer/packages/rendering/src/glsl/sector/primitives/sphericalSegment.vert

```diff
@@ -4,29 +4,29 @@
 uniform mat4 inverseModelMatrix;
 uniform mat4 inverseNormalMatrix;
 
-attribute float a_treeIndex;
-attribute vec3 a_color;
-attribute vec3 a_center;
-attribute vec3 a_normal;
-attribute float a_horizontalRadius;
-attribute float a_verticalRadius;
-attribute float a_height;
-
-varying float v_treeIndex;
+in float a_treeIndex;
+in vec3 a_color;
+in vec3 a_center;
+in vec3 a_normal;
+in float a_horizontalRadius;
+in float a_verticalRadius;
+in float a_height;
+
+out float v_treeIndex;
 // We pack vRadius as w-component of center
-varying vec4 center;
-varying float hRadius;
-varying float height;
+out vec4 center;
+out float hRadius;
+out float height;
 
 // U, V, axis represent the 3x3 sphere basis.
 // They are vec4 to pack extra data into the w-component
-// since Safari on iOS only supports 8 varying vec4 registers.
-varying vec4 U;
-varying vec4 V;
-varying vec4 sphereNormal;
+// since Safari on iOS only supports 8 out vec4 registers.
```

Comment:
This restriction seems to be outdated, WebGL should support at least 16 attributes: https://webgl2fundamentals.org/webgl/lessons/webgl-attributes.html I think we can remove this comment.

Reply:
This is true, but I wanted these comments, so that we will have reference to make changes for future

Reply:
Yes, alright - I guess it's not that important anyway